runners/gearpump/src/main/java/org/apache/beam/runners/gearpump/GearpumpRunner.java

```diff
@@ -69,15 +98,23 @@ public static GearpumpRunner fromOptions(PipelineOptions options) {
 
   public <OutputT extends POutput, InputT extends PInput> OutputT apply(
       PTransform<InputT, OutputT> transform, InputT input) {
-    if (Flatten.FlattenPCollectionList.class.equals(transform.getClass())
+    if (overrides.containsKey(transform.getClass())) {
```

Comment:
The next time you sync from master, this method will go away and you'll need to do all of this via `runners-core-construction` utilities. So let's definitely do this PR first so you don't have too much stuff going on at once.